Source/AH/Mixins/Throttling.lua

```diff
@@ -0,0 +1,50 @@
+AuctionatorAHThrottlingFrameMixin = {}
+
+local Mixin = AuctionatorAHThrottlingFrameMixin
+
+local THROTTLING_EVENTS = {
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_DROPPED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_QUEUED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_RESPONSE_RECEIVED",
+  "AUCTION_HOUSE_THROTTLED_MESSAGE_SENT",
+  "AUCTION_HOUSE_THROTTLED_SPECIFIC_SEARCH_READY",
+  "AUCTION_HOUSE_THROTTLED_SYSTEM_READY",
+  "AUCTION_HOUSE_BROWSE_FAILURE"
+}
+
+function Mixin:OnLoad()
```

Comment:
Can we keep the use of the explicit Mixin name defined above? I prefer not to have to go through a level of naming indirection to understand the implementation, and Mixin doesn't add any clarity for me.